ignite/templates/app/stargate/readme.md.plush

```diff
@@ -1,29 +1,29 @@
 # <%= AppName %>
```

Comment:
oops what is stargate and why do we have their readme here?

Reply:
Cosmos SDK v0.39.x was codename "Launchpad" and v0.40.x was codename "Stargate". At some point through the transitionary period, we supported both. Later dropped support for the old version, but "Stargate" is still in the codebase.

Reply:
@fadeev does this file require updates?